megamek/src/megamek/server/GameManager.java

```diff
@@ -20451,7 +20451,8 @@ private Vector<Report> resolveCrewDamage(Entity e, int damage, int crewPos) {
                     // return true;
                 } // else
                 vDesc.add(r);
-            } while (e.shouldUseEdge(OptionsConstants.EDGE_WHEN_KO)
+            } while ((e.shouldUseEdge(OptionsConstants.EDGE_WHEN_KO)
+                    && e.getCrew().isKoThisRound(crewPos))
```

Comment:
This check was accidentally removed when refactoring, which led to entities rerolling edge until the pilot blacked out.

Reply:
NGL, the effect was hilarious.  But good find!